### PR TITLE
fixed linting

### PR DESCRIPTION
**Pull request by TitusEntertainment**
Are you happy now @binaryoverload?

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -38,7 +38,10 @@
 				"no-dupe-class-members": "off",
 				"require-atomic-updates": "off",
 				"newline-per-chained-call": "off",
-				"@typescript-eslint/indent": ["off", 4]
+				"no-tabs": 1,
```
Comment (binaryoverload) [inline .eslintrc.json:41]:
```suggestion
				"no-tabs": 2,
```

File: .eslintrc.json [hunk 2]
```diff
@@ -38,7 +38,10 @@
...
+				"no-mixed-spaces-and-tabs": 1,
```
Comment (binaryoverload) [inline .eslintrc.json:42]:
```suggestion
				"no-mixed-spaces-and-tabs": 2,
```

**Review by binaryoverload — CHANGES_REQUESTED** [review]
I'd be happy now 😂

**Comment by binaryoverload** [comment]
Also you might want to create an editor config!

**Comment by TitusEntertainment** [comment]
Fine....

**Comment by TitusEntertainment** [comment]
Also, I do have an editor config but it's local and .gitignored

**Pull request merged**